draft-ietf-tls-esni.md

```diff
@@ -438,7 +294,7 @@ structure:
        ServerESNIResponseType response_type;
        select (response_type) {
            case esni_accept:        uint8 nonce[16];
-           case esni_retry_request: ESNIKeys retry_keys<1..2^16-1>;
+           case esni_retry_request: ESNIConfig retry_keys<1..2^16-1>;
```

Comment:
Nit: s/retry_keys/retry_configs/

Reply:
Benefit of the ESNIConfig rename: really `retry_keys` should have been `retry_keyses`, but now we don't have to pretend to be Gollum. :-)